upsetplot/data.py

```diff
@@ -16,3 +19,51 @@ def generate_data(seed=0, n_samples=10000, n_sets=3, aggregated=False):
     if aggregated:
         return df.value.groupby(level=list(range(n_sets))).count()
     return df.value
+
+
+def from_sets(data):
+    """
+    Data loader for a dict of sets
+
+    :param Dict[str, set[str]] data:
+    :return: Multi-Index Series of intersection counts
+    :rtype: pd.Series
+    """
+    # Convert dict into OrderedDict to preserve Key/Value order
+    data = OrderedDict(data)
+
+    # Construct Index
+    tf_array = [[True, False]] * len(data)
+    index = pd.MultiIndex.from_product(tf_array, names=data.keys())
+
+    # Curate values from each intersection group
+    values = []
+    for i in index:
+        values.append(_intersection_counts(data.values(), i))
+
+    return pd.Series(values, index=index)
+
+
+def _intersection_counts(sets, bool_tuple):
+    """
+    Given list of sets and boolean tuple, return count of intersection
+
+    :param List[sets[str]] sets:
```

Comment:
Consistent with matplotlib and pandas, we use the [numpydoc docstring conventions](https://numpydoc.readthedocs.io/en/latest/format.html). I know it's more verbose, and maybe I'll change in the future, but for now, we should follow the conventions that this sphinx instance is set up to render.

Reply:
Not a problem, I should've looked at the docstring convention in `plotting.py` first.